spiderfoot/correlation.py

```diff
@@ -56,33 +70,54 @@ def __init__(self, dbh: SpiderFootDb, ruleset: dict, scanId: str = None):
             try:
                 self.rules.append(yaml.safe_load(ruleset[rule_id]))
                 self.rules[len(self.rules) - 1]['rawYaml'] = ruleset[rule_id]
-                # Strip any extra newlines that may have creeped into meta
-                for rule in self.rules:
-                    for k in rule['meta'].keys():
-                        if isinstance(rule['meta'][k], str):
-                            rule['meta'][k] = rule['meta'][k].strip()
-                        else:
-                            rule['meta'][k] = rule[k]
-                continue
```

Comment:
Moved this out of the loop as there's no need to check the entirety of `self.rules` upon every iteration - it can wait until after the loop completes. Also the `continue` on the last line of the loop was redundant.